storage/interfaces.go

```diff
@@ -1,31 +1,33 @@
-package store
+package storage
 
 // NoSizeLimit is represented as -1 for arguments to GetMeta
 const NoSizeLimit int64 = -1
 
 // MetadataStore must be implemented by anything that intends to interact
 // with a store of TUF files
 type MetadataStore interface {
-	GetMeta(name string, size int64) ([]byte, error)
-	SetMeta(name string, blob []byte) error
-	SetMultiMeta(map[string][]byte) error
+	GetSized(name string, size int64) ([]byte, error)
+	Set(name string, blob []byte) error
+	SetMulti(map[string][]byte) error
 	RemoveAll() error
-	RemoveMeta(name string) error
+	Remove(name string) error
```

Comment:
Wondering if `Location` should be added too, since all of the implementations all also provide a `Location` function?